index.js

```diff
@@ -170,75 +169,83 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
 
 /**
  * ### Challenge `getFullNames`
- * 
+ *
  * @instructions
  * Implement this function using forEach().
- * 
+ *
  * @param runners array of runners like the one inside the /data/runners.js file.
  * @returns an array with all the runners' full names in the following format: "Smith, John".
  * The full names appear in the array in the same order the runners appear in the `runners` array.
 */
-function getFullNames(/* CODE HERE */) {
-  /* CODE HERE */
+function getFullNames(runners) {
+  const fullNames = [];
+  runners.map(runners => fullNames.push(`${runners.last_name}, ${runners.first_name}`));
+  return fullNames;
 }
```

Comment:
Remember that .map create a new array for you, so you don't have to manually make an new array and then push the items into that array.  You are basically creating two arrays here.

Reply:
Also, look at line 174 it says to use .forEach (when using forEach instead of map, you will need to push the results into a new array.